vastloader.go

```diff
@@ -0,0 +1,59 @@
+package vast
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"time"
+)
+
+type VASTResponse struct {
+	Body []byte
+	Err  error
+}
+
+// VASTResponseHandler handles the repsonse from the request
+func VASTResponseHandler(res *http.Response, err error) VASTResponse {
```

Comment:
```suggestion
func ResponseHandler(res *http.Response, err error) ([]byte, error) {
```
I'm not sure we need the `VASTResponse` struct at all if we return the byte slice and error separately.

We can also omit the `VAST` prefix here as we're already in the `vast` package.